templates/default.nginx_http.conf.template

```diff
@@ -25,6 +25,20 @@ add_header X-Frame-Options DENY;
 # to make sure they are authentic. This will become obsolete in June 2021.
 add_header Expect-CT 'enforce, max-age=604800';
 
+# Enable cross-site scripting filter (XSS)
+# (handled by the browser; usually enabled by default,
+# this header will re-enable the filter if disabled by user.)
+add_header X-XSS-Protection "1; mode=block";
+
+# Explicitly list domains allowed to serve content for this site
+add_header Content-Security-Policy-Report-Only "default-src 'self'; object-src 'none'; base-uri 'self'; manifest-src 'self'; media-src 'self'; worker-src 'none';";
+add_header Content-Security-Policy-Report-Only "script-src 'report-sample' 'self' https://discuss.dockstore.org https://gui.dockstore.org https://platform.twitter.com https://www.google-analytics.com https://www.googletagmanager.com;";
+add_header Content-Security-Policy-Report-Only "style-src 'report-sample' 'self' https://cdnjs.cloudflare.com https://fonts.googleapis.com https://gui.dockstore.org;";
+add_header Content-Security-Policy-Report-Only "connect-src 'self' https://s3.amazonaws.com https://api.github.com https://view.commonwl.org;";
```

Comment:
The line to accept s3 leads me to consider that we ought to put cloudfront/DNS in front of our featured content bucket. If I understand correctly having a `connect-src` policy that allows S3 would allow any JavaScript on S3 to pass the policy.

Reply:
there is a securityhub alarm that this might help with too
s3-bucket-public-read-prohibited is tripped because we serve read-only http content from it, but putting cloudfront in front might help